diary-alexander-jaffray/modernized/15-notes-diary.adoc

```diff
@@ -1056,24 +1093,27 @@ and to use such plainness of speech.
 Therefore we hope we shall not be mistaken; yet if the Lord, by you or yet any other,
 will hold forth convincing light to us, discovering our mistakes,
 we should gladly close with it; only,
-if we be necessitated to take any course for expeding +++[+++ridding]
+if we be necessitated to take any course for _expeding_ +++[+++ridding]
 ourselves from the present snares and guiltiness, under which we lie,
 we desire you will not construe it hardly.
 
 Now, that light may be let forth to you and us both,
 from Him who is the Father of lights,
 shall be the earnest desire of your loving brethren in Christ,
 
-In name of others in this place, sic subscribitur,
+[.signed-section-closing]
+In name of others in this place, _sic subscribitur,_
 
-Alexander Jaffray, Mr. William Moore, Mr. John Row, Mr. John Meinzies, Andrew Birnie.`"
+[.signed-section-signature]
+Alexander Jaffray, Mr. William Moore, Mr. John Row, Mr. John Meinzies, Andrew Birnie.
 
-=== Note S.
+--
 
-The following quotation from Orme`'s Life of Owen, p. 127,
+=== Note S.
+The following quotation from Orme`'s [.book-title]#Life of Owen,# p. 127,
 may throw some light on these reflections of Jaffray.
 
-The state of religion in Scotland, during the ten years preceding the English invasion,
+"`The state of religion in Scotland, during the ten years preceding the English invasion,
```

Comment:
This is a quote with quotations at the end, but I wasn't sure where it began but from the paragraph above it, it seems to start right away. But idk if that's right.

Reply:
yeah, I think you're correct.